.github/ISSUE_TEMPLATE/Release_23.12_FOSS_ComplianceTemplate.md

```diff
@@ -0,0 +1,80 @@
+---
+name: Release Compliance 23.12
+about: Use this template for tracking Catena-X Release 23.12.
+title: "[FOSS NAME] Release Compliance"
+labels: compliance
+assignees: @kelaja
+milestones: 23.12
+---
+
+<!-- 
+Thanks for your contribution! Please fill out this template as good as possible. 
+Important: Contributing Guidelines can be found here: https://eclipse-tractusx.github.io/docs/oss/how-to-contribute
+Checkout the repository README for process description. 
+-->
+
+
+
+## Release Compliance 23.12
+Source in Catena-X Confluence + Expert Contacts [here](https://confluence.catena-x.net/x/DOZkBQ).
+
+
+ - [ ] **Gaia-X compliance** confirmed by Catena-X expert.
```

Comment:
If I am not mistaken, we should NOT use "Catena-X" or "CX" within the boundaries of Tractus-X => I guess this is valid in general for legal/naming conventions.
So here this would result in "confirmed by (dedicated) Expert."